debezium-core/src/main/java/io/debezium/util/DelayStrategy.java

```diff
@@ -102,12 +106,14 @@ public boolean sleepWhen(boolean criteria) {
      * Create a delay strategy that applies an exponentially-increasing delay as long as the criteria is met. As soon as
      * the criteria is not met, the delay resets to zero.
      *
-     * @param initialDelayInMilliseconds the initial delay; must be positive
-     * @param maxDelayInMilliseconds the maximum delay; must be greater than the initial delay
+     * @param InitialDuration the initial delay; must be positive
+     * @param MaxDuration the maximum delay; must be greater than the initial delay
      * @return the strategy; never null
      */
-    public static DelayStrategy exponential(long initialDelayInMilliseconds, long maxDelayInMilliseconds) {
-        return exponential(initialDelayInMilliseconds, maxDelayInMilliseconds, 2.0);
+    public static DelayStrategy exponential(Duration InitialDuration, Duration MaxDuration) {
```

Comment:
Please use names `initialDelay`, `maxDelay`. No new vars needed, can be inlined.